apkexporter/apkexporter_test.go

```diff
@@ -68,11 +68,11 @@ func Test_prepareKeystoreConfig_SuccessDownload(t *testing.T) {
 	exporter := givenExporter(mockAPKBuilder, mockFileDownloader)
 
 	// When
-	output, err := exporter.prepareKeystoreConfig(givenKeystoreConfig("http://url.com/keystore.keystore"))
+	output, err := exporter.prepareKeystoreConfig(givenKeystoreConfig("http://url.com/keystore.jks"))
 
 	// Then
 	require.NoError(t, err)
-	require.Contains(t, output.Path, "keystore.keystore")
+	require.Contains(t, output.Path, "keystore.jks")
```

Comment:
Incase a keystore is provided via URL, it will be downloaded to a temp directory. This test only ensures that the filename is not changed but it does not care about where it is downloaded. To reflect this more I will change the assertion to explicitly check the file name and not only check for contains.